content/applications/studio/models_modules_apps.rst

```diff
@@ -212,16 +212,84 @@ Export and import customizations
+   export and import<../essentials/export_import_data>` functions, means data is imported in a
+   logical way. For example, if the module contains customers and sales orders, the customers are
+   created first, since these are required for the sales orders to be created.
+
+.. _studio/export-import/export:
+
+Export customizations
+---------------------
+
+To export customizations, click the :icon:`oi-studio` :guilabel:`(Toggle Studio)` button on the main
+Odoo dashboard, then :guilabel:`Export`, then either:
+
+- download all Studio customizations by clicking the :guilabel:`Export` button; or
+- choose what data to export by clicking :ref:`Configure data and demo data to export
+  <studio/export-import/export/configure>`.
+
+.. _studio/export-import/export/configure:
+
+Configure data to export
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+To select specific models to export, click :guilabel:`New` on the :guilabel:`Studio Export` screen,
+then start typing the name of the relevant model or select it from the list.
+
+.. tip::
+
+   Click :guilabel:`Preset` to see a list of all models in your database with records that have
+   been modified using Studio and all custom models created using Studio. To configure one of these
+   models for export, click on the model to open it and make the required changes.
+
+Tick the following options as relevant:
+
+- :guilabel:`Demo`: if the exported records should be considered as demo data when imported.
+- :guilabel:`Attachments`: if attachments related to exported records should be included in the
+  export.
+- :guilabel:`Updatable`: if the exported records should be able to be updated during a module update.
 
-To import and install these customizations in another database, connect to the destination database
-and go to :menuselection:`Main dashboard --> Studio --> Customizations --> Import`, then upload
-the exported ZIP file before clicking on the :guilabel:`Import` button.
+If necessary, edit the :guilabel:`Domain` to determine which of the model's records should be
+exported. To do so, click the :guilabel:`Edit Domain` button or :icon:`fa-caret-right`
+:guilabel:`(Modify filter)` then :guilabel:`Edit Domain`, as appropriate. Proceed to make any
+required changes.
+
+After configuring a model for export, click :guilabel:`Studio Export` to return to the main screen.
+To download a ZIP file with the customizations for all the listed models, click :guilabel:`Export`.
+
+.. note::
+   It is not necessary to select one or more models as all listed models will be included in the
+   export. To remove a model from the export, select it and click the :icon:`fa-cog`
+   :guilabel:`Actions` button then :icon:`fa-trash-o` :guilabel:`Delete`.
+
+Choose whether to only :guilabel:`Include Data` or to also
+:guilabel:`Include Demo Data`, then click the :guilabel:`Export` button.
+
+.. image:: models_modules_apps/studio-export.png
+      :alt: Choosing to export both data and demo data
```

Comment:
```suggestion
   :alt: Choosing to export both data and demo data
```
The `:alt:` markup is misaligned with `.. image:::` (it should still work, but it's our "coding style")